internal/pkg/earpointer/analysis.go

```diff
@@ -550,7 +615,7 @@ func (vis *visitor) unifyLocals(v1 ssa.Value, v2 ssa.Value) {
 		return
 	}
 	state := vis.state
-	for _, context := range vis.contexts {
+	for _, context := range vis.getContexts(v1) {
```

Comment:
`u` and `v` are better names than `v1` and `v2`. Shorter and convey the same info.